DetailedSoldierListWOTC/Src/DetailedSoldierListWOTC/Classes/X2EventListener_DetailedSoldierList.uc

```diff
@@ -0,0 +1,73 @@
+// X2EventListener_DetailedSoldierList.uc
+// 
+// A listener template that integrates with Community Highlander events to
+// customize how soldier statuses are displayed.
+//
+class X2EventListener_DetailedSoldierList extends X2EventListener;
+
+static function array<X2DataTemplate> CreateTemplates()
+{
+	local array<X2DataTemplate> Templates;
+
+	Templates.AddItem(CreateStatusListeners());
+
+	return Templates;
+}
+
+static function CHEventListenerTemplate CreateStatusListeners()
+{
+	local CHEventListenerTemplate Template;
```

Comment:
This will crash if CHL isn't loaded. I suggest adding a check before `CreateStatusListeners()` call